UKHomeOffice/cerberus-service
# COP-11821 - Port field

## Description
This PR moves the `port` field from the **Target information** section to the **Movement details** section.

The port field text is also alternated based on the movement direction.
<img width="924" alt="image" src="https://user-images.githubusercontent.com/19333750/187741826-72fa4da9-f7e6-4874-b4d9-801310f0e5a9.png">

<img width="902" alt="image" src="https://user-images.githubusercontent.com/19333750/187741863-8de0269b-1a6a-4277-ad8a-13c26d7e7475.png">

### Supplemental
This PR also fixes a bug found where after a target has been issued, the notes form is still displayed.


## To Test
- Pull & run against dev.
- Claim a task.
- Click on the `Issue target` target.
- Click on the `Change` link within the movement area.
- Select the movement direction radio button and the port field should display with the appropriate hint/description text.
